src/lib/CloudServiceCard.svelte

```diff
@@ -45,6 +46,18 @@ import { trim } from './util';
 export let service: CloudService;
 let metricConfigurations: Map<string, MetricConfiguration> = new Map<string, MetricConfiguration>();
 
+async function remove(target: string) {
+	const ok = confirm('Do you really want to delete this cloud service?');
+	if (!ok) {
+		return;
+	}
+
+	await removeCloudService(target);
+
+	// invalidate the cloud service
+	invalidate(`/v1/orchestrator/cloud_services/${target}`);
```

Comment:
invalidate won't work in a component svelte, it works only in route files. You need to dispatch an event from the component here, "listen" to the event on the route file and then call remove/invalidate. There should be code that does things like that in the ToE components

Reply:
I did it like this, did I forget something?  https://github.com/clouditor/clouditor-ui/blob/9f55da2c457055099c3d6c23339306205f1d183e/src/routes/cloud/%5B...id%5D/configuration/toe/%2Bpage.svelte#L35

Reply:
You need to dispatch the event from the component to the route